sled-agent/src/http_entrypoints.rs

```diff
@@ -290,7 +290,10 @@ impl SledAgentApi for SledAgentImpl {
             file,
         } = path_params.into_inner();
 
-        let range = rqctx.range();
```

Comment:
Same as for the Nexus APIs - the sled agent already knew how to process these range requests, it just wasn't part of the openapi spec.